src/AccessibilityInsights.SharedUx/Controls/SettingsTabs/AboutTabControl.xaml

```diff
@@ -24,8 +24,8 @@
                 <Run Text="{Binding Path=VersionInfoLabel, Mode=OneWay}" />
             </Hyperlink>
         </TextBlock>
-        <Label Content="{Binding Path=AxeVersion}" ContentStringFormat="{x:Static Properties:Resources.AboutTabControl_AxeVersionFormat}"
-               Style="{StaticResource lblDark}" Margin="0,5,0,0"/>
+        <TextBlock Text="{Binding Path=AxeVersion, StringFormat={x:Static Properties:Resources.AboutTabControl_AxeVersionFormat}}"
+               Style="{StaticResource tbLine}" Margin="0,5,0,0"/>
```

Comment:
lblDark inherits a LineHeight setting of `DynamicResource StandardTextSize`, while tbLine has a hardcoded LineHeight of 10. That feels like it might cause visual issues with different font sizes. Should we create a new style that inherits fro `VerStandardTextBlock` but doesn't change the LineHeight?

Reply:
updated, there was an existing style